### ISSUE TITLE
[tests] Periodic test failure after switch from zuul to GHA

### ISSUE THREAD

**Issue by cidrblock** [ansible/ansible-navigator#812]
Please log them here with URL, PR number, and summary information when they occur so a trend can be found if one exists

Related #758

**Comment by cidrblock** [comment]
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[9-:open {{ task_path }}-goto vi]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[6-:doc-doc for task]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[8-:back-show doc]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[9-:back-show task]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[10-:open {{ task_path }}-goto vi]

(PR unknown)

**Comment by ssbarnea** [comment]
```
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[7-:back-show doc]

```

**Comment by cidrblock** [comment]
In #814 
```
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[9-:open {{ task_path }}-goto vi]
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[9-:open {{ task_path }}-goto vi]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[8-:back-show doc]
```
```
=================================== FAILURES ===================================
__________________ Test.test[9-:open {{ task_path }}-goto vi] __________________
[gw0] linux -- Python 3.10.2 /home/runner/work/ansible-navigator/ansible-navigator/.tox/py310/bin/python

self = <tests.integration.actions.templar.test_direct_interactive_ee.Test object at 0x7f0910292dd0>
request = <FixtureRequest for <Function test[9-:open {{ task_path }}-goto vi]>>
tmux_session = <tests.integration._tmux_session.TmuxSession object at 0x7f0910291bd0>
step = Step(user_input=':open {{ task_path }}', comment='goto vi', look_fors=['---'], look_nots=[], mask=True, playbook_status=None, step_index=9, search_within_response='---')

    def test(self, request, tmux_session, step):
        # pylint: disable=too-many-branches
        # pylint: disable=too-many-locals
        """test interactive and ``stdout`` mode ``config``"""
    
        if step.search_within_response is SearchFor.HELP:
            search_within_response = ":help help"
        elif step.search_within_response is SearchFor.PROMPT:
            search_within_response = tmux_session.cli_prompt
        else:
            search_within_response = step.search_within_response
    
        received_output = tmux_session.interaction(
            value=step.user_input,
            search_within_response=search_within_response,
        )
    
        if step.mask:
            # mask out some configuration that is subject to change each run
            mask = "X" * 50
            for idx, line in enumerate(received_output):
                if tmux_session.cli_prompt in line:
                    received_output[idx] = mask
                else:
                    for out in ["duration:", "playbook:", "start:", "end:", "task_path:"]:
                        if out in line:
                            received_output[idx] = mask
    
        fixtures_update_requested = (
            self.UPDATE_FIXTURES
            or os.environ.get("ANSIBLE_NAVIGATOR_UPDATE_TEST_FIXTURES") == "true"
        )
        if fixtures_update_requested:
            update_fixtures(
                request,
                step.step_index,
                received_output,
                step.comment,
                additional_information={
                    "look_fors": step.look_fors,
                    "look_nots": step.look_nots,
                    "compared_fixture": not any((step.look_fors, step.look_nots)),
                },
            )
        page = " ".join(received_output)
    
        if step.look_fors:
>           assert all(look_for in page for look_for in step.look_fors)
E           AssertionError

fixtures_update_requested = False
idx        = 0
line       = ('(py310) bash-5.0$  ansible-navigator run '
 '/home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/run/site.yaml '
 '-i '
 '/home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/run/inventory '
 '--ee True --ll debug --mode interactive')
mask       = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
page       = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
received_output = ['XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX']
request    = <FixtureRequest for <Function test[9-:open {{ task_path }}-goto vi]>>
search_within_response = '---'
self       = <tests.integration.actions.templar.test_direct_interactive_ee.Test object at 0x7f0910292dd0>
step       = Step(user_input=':open {{ task_path }}', comment='goto vi', look_fors=['---'], look_nots=[], mask=True, playbook_status=None, step_index=9, search_within_response='---')
tmux_session = <tests.integration._tmux_session.TmuxSession object at 0x7f0910291bd0>

tests/integration/actions/templar/base.py:119: AssertionError
__________________ Test.test[9-:open {{ task_path }}-goto vi] __________________
[gw0] linux -- Python 3.10.2 /home/runner/work/ansible-navigator/ansible-navigator/.tox/py310/bin/python

self = <tests.integration.actions.templar.test_direct_interactive_noee.Test object at 0x7f0910445ff0>
request = <FixtureRequest for <Function test[9-:open {{ task_path }}-goto vi]>>
tmux_session = <tests.integration._tmux_session.TmuxSession object at 0x7f0910447bb0>
step = Step(user_input=':open {{ task_path }}', comment='goto vi', look_fors=['---'], look_nots=[], mask=True, playbook_status=None, step_index=9, search_within_response='---')

    def test(self, request, tmux_session, step):
        # pylint: disable=too-many-branches
        # pylint: disable=too-many-locals
        """test interactive and ``stdout`` mode ``config``"""
    
        if step.search_within_response is SearchFor.HELP:
            search_within_response = ":help help"
        elif step.search_within_response is SearchFor.PROMPT:
            search_within_response = tmux_session.cli_prompt
        else:
            search_within_response = step.search_within_response
    
        received_output = tmux_session.interaction(
            value=step.user_input,
            search_within_response=search_within_response,
        )
    
        if step.mask:
            # mask out some configuration that is subject to change each run
            mask = "X" * 50
            for idx, line in enumerate(received_output):
                if tmux_session.cli_prompt in line:
                    received_output[idx] = mask
                else:
                    for out in ["duration:", "playbook:", "start:", "end:", "task_path:"]:
                        if out in line:
                            received_output[idx] = mask
    
        fixtures_update_requested = (
            self.UPDATE_FIXTURES
            or os.environ.get("ANSIBLE_NAVIGATOR_UPDATE_TEST_FIXTURES") == "true"
        )
        if fixtures_update_requested:
            update_fixtures(
                request,
                step.step_index,
                received_output,
                step.comment,
                additional_information={
                    "look_fors": step.look_fors,
                    "look_nots": step.look_nots,
                    "compared_fixture": not any((step.look_fors, step.look_nots)),
                },
            )
        page = " ".join(received_output)
    
        if step.look_fors:
>           assert all(look_for in page for look_for in step.look_fors)
E           AssertionError

fixtures_update_requested = False
idx        = 0
line       = ('(py310) bash-5.0$  ansible-navigator run '
 '/home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/run/site.yaml '
 '-i '
 '/home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/run/inventory '
 '--ee False --ll debug --mode interactive')
mask       = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
page       = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
received_output = ['XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX']
request    = <FixtureRequest for <Function test[9-:open {{ task_path }}-goto vi]>>
search_within_response = '---'
self       = <tests.integration.actions.templar.test_direct_interactive_noee.Test object at 0x7f0910445ff0>
step       = Step(user_input=':open {{ task_path }}', comment='goto vi', look_fors=['---'], look_nots=[], mask=True, playbook_status=None, step_index=9, search_within_response='---')
tmux_session = <tests.integration._tmux_session.TmuxSession object at 0x7f0910447bb0>

tests/integration/actions/templar/base.py:119: AssertionError
_________________________ Test.test[8-:back-show doc] __________________________
[gw0] linux -- Python 3.10.2 /home/runner/work/ansible-navigator/ansible-navigator/.tox/py310/bin/python

self = <tests.integration.actions.templar.test_welcome_interactive_ee.Test object at 0x7f091043ad10>
request = <FixtureRequest for <Function test[8-:back-show doc]>>
tmux_session = <tests.integration._tmux_session.TmuxSession object at 0x7f09104395d0>
step = Step(user_input=':back', comment='show doc', look_fors=['module: debug'], look_nots=[], mask=True, playbook_status=None, step_index=8, search_within_response=<SearchFor.HELP: 'search for help'>)

    def test(self, request, tmux_session, step):
        # pylint: disable=too-many-branches
        # pylint: disable=too-many-locals
        """test interactive and ``stdout`` mode ``config``"""
    
        if step.search_within_response is SearchFor.HELP:
            search_within_response = ":help help"
        elif step.search_within_response is SearchFor.PROMPT:
            search_within_response = tmux_session.cli_prompt
        else:
            search_within_response = step.search_within_response
    
        received_output = tmux_session.interaction(
            value=step.user_input,
            search_within_response=search_within_response,
        )
    
        if step.mask:
            # mask out some configuration that is subject to change each run
            mask = "X" * 50
            for idx, line in enumerate(received_output):
                if tmux_session.cli_prompt in line:
                    received_output[idx] = mask
                else:
                    for out in ["duration:", "playbook:", "start:", "end:", "task_path:"]:
                        if out in line:
                            received_output[idx] = mask
    
        fixtures_update_requested = (
            self.UPDATE_FIXTURES
            or os.environ.get("ANSIBLE_NAVIGATOR_UPDATE_TEST_FIXTURES") == "true"
        )
        if fixtures_update_requested:
            update_fixtures(
                request,
                step.step_index,
                received_output,
                step.comment,
                additional_information={
                    "look_fors": step.look_fors,
                    "look_nots": step.look_nots,
                    "compared_fixture": not any((step.look_fors, step.look_nots)),
                },
            )
        page = " ".join(received_output)
    
        if step.look_fors:
>           assert all(look_for in page for look_for in step.look_fors)
E           AssertionError

fixtures_update_requested = False
mask       = 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
page       = ''
received_output = []
request    = <FixtureRequest for <Function test[8-:back-show doc]>>
search_within_response = ':help help'
self       = <tests.integration.actions.templar.test_welcome_interactive_ee.Test object at 0x7f091043ad10>
step       = Step(user_input=':back', comment='show doc', look_fors=['module: debug'], look_nots=[], mask=True, playbook_status=None, step_index=8, search_within_response=<SearchFor.HELP: 'search for help'>)
tmux_session = <tests.integration._tmux_session.TmuxSession object at 0x7f09104395d0>

tests/integration/actions/templar/base.py:119: AssertionError
```

**Comment by webknjaz** [comment]
Just saw these flaky failures in PR #813:

_(Python 3.9, https://github.com/ansible/ansible-navigator/runs/4970145619?check_suite_focus=true#step:7:4218)_
```console
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[5-:doc-doc for task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[7-:back-show doc]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[8-:back-show task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[9-:open {{ task_path }}-goto vi]
```

_(Python 3.10, https://github.com/ansible/ansible-navigator/runs/4970145702?check_suite_focus=true#step:7:3714)_
```console
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[7-:back-show doc]
```

**Comment by cidrblock** [comment]
In #815 

(python 3.10 https://github.com/ansible/ansible-navigator/runs/4978189330?check_suite_focus=true#step:7:3269)

```
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[9-:open {{ task_path }}-goto vi]
```

**Comment by cidrblock** [comment]
In #819 

(py 3.10 https://github.com/ansible/ansible-navigator/runs/4981223787?check_suite_focus=true#step:7:3269)

```
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[5-:doc-doc for task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[7-:back-show doc]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[8-:back-show task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[9-:open {{ task_path }}-goto vi]
```

**Comment by webknjaz** [comment]
Python 3.6:

_(https://github.com/ansible/ansible-navigator/runs/4986508790?check_suite_focus=true)_
```console
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[9-:open {{ task_path }}-goto vi]
```

**Comment by cidrblock** [comment]
The below is all after #823 was merged

**Comment by cidrblock** [comment]
In #817 
(py 3.10 https://github.com/ansible/ansible-navigator/runs/4986990365?check_suite_focus=true#step:7:3277)
[```console
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[5-:doc-doc for task]
```](https://github.com/ansible/ansible-navigator/runs/4986990365?check_suite_focus=true#step:7:3277)

[](url)

**Comment by cidrblock** [comment]
In #799

(py 3.10 https://github.com/ansible/ansible-navigator/runs/4987450501?check_suite_focus=true)

FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[6-:doc-doc for task]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[8-:back-show doc]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[9-:back-show task]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[10-:open {{ task_path }}-goto vi]
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[11-:q!-exit vi]

**Comment by cidrblock** [comment]
In #836
(py38 https://github.com/ansible/ansible-navigator/runs/4991446900?check_suite_focus=true)
```console
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[5-:doc-doc for task]
```

**Comment by cidrblock** [comment]
In #818 

(py3.10 https://github.com/ansible/ansible-navigator/runs/4991856331?check_suite_focus=true)

```console
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[5-:doc-doc for task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[5-:doc-doc for task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[7-:back-show doc]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[8-:back-show task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[9-:open {{ task_path }}-goto vi]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[10-:q!-exit vi]
```

**Comment by cidrblock** [comment]
In #832 

(py3.10 https://github.com/ansible/ansible-navigator/runs/4990902605?check_suite_focus=true)

```console
FAILED tests/integration/actions/templar/test_welcome_interactive_ee.py::Test::test[8-:back-show doc]
```

**Comment by cidrblock** [comment]
In #842 

(py3.10 https://github.com/ansible/ansible-navigator/runs/5007225980?check_suite_focus=true)
```console
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[7-:back-show doc]
```

**Comment by cidrblock** [comment]
In #845 

(py3.10 https://github.com/ansible/ansible-navigator/runs/5008653434?check_suite_focus=true)

```console
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[5-:doc-doc for task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[5-:doc-doc for task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[7-:back-show doc]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[8-:back-show task]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[9-:open {{ task_path }}-goto vi]
FAILED tests/integration/actions/templar/test_direct_interactive_ee.py::Test::test[10-:q!-exit vi]
```

**Comment by cidrblock** [comment]
In #846 

(py3.7 https://github.com/ansible/ansible-navigator/runs/5008889493?check_suite_focus=true)

```console
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[5-:doc-doc for task]
```

**Comment by cidrblock** [comment]
In #847 

(py3.8 https://github.com/ansible/ansible-navigator/runs/5009241907?check_suite_focus=true)
```console
FAILED tests/integration/actions/templar/test_welcome_interactive_noee.py::Test::test[6-:doc-doc for task]
```

(py3.10 https://github.com/ansible/ansible-navigator/runs/5009242064?check_suite_focus=true)
```
FAILED tests/integration/actions/templar/test_direct_interactive_noee.py::Test::test[7-:back-show doc]
FAILED tests/integration/actions/templar/test_welcome_interactive_noee.py::Test::test[6-:doc-doc for task]
```

**Comment by cidrblock** [comment]
The below occured after #854

**Comment by cidrblock** [comment]
In #857 

(py3.6 https://github.com/ansible/ansible-navigator/runs/5036171177?check_suite_focus=true)

```console
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[0-ansible-navigator --execution-environment true-welcome-:help help]
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[1-:replay /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/replay/playbook-artifact.json-Play list-search_within_response1]
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[2-:0-Task list-:help help]
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[3-:0-Task 1-:help help]
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[4-:stdout-Check stdout-:help help]
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[5-:back-Return to task 1-:help help]
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[6-:back-Return to task list-:help help]
ERROR tests/integration/actions/replay/test_welcome_interactive_ee.py::Test::test[7-:back-Return to play list-:help help]
```

**Comment by cidrblock** [comment]
In #858 

(py3.8 https://github.com/ansible/ansible-navigator/runs/5054614457?check_suite_focus=true)

```console
ERROR tests/integration/actions/doc/test_welcome_interactive_ee.py::TestModuleDocNotExist::test[0-ansible-navigator --execution-environment true-welcome-module_doc_fail-expected_in_output0]
ERROR tests/integration/actions/doc/test_welcome_interactive_ee.py::TestModuleDocNotExist::test[1-:doc testorg.coll_1.doesnotexist-ansible-navigator doc wrong plugin name-module_doc_fail-expected_in_output1]
```

**Comment by cidrblock** [comment]
In #864 

(py3.9 https://github.com/ansible/ansible-navigator/runs/5054608102?check_suite_focus=true)

```console
FAILED tests/integration/actions/config/test_stdout_tmux.py::Test::test[config specified configuration file without ee  clear && ansible-navigator config dump -c /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/config/ansible.cfg --ee False --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
the following occured after #867

**Comment by cidrblock** [comment]
in #870 

(py3.10 https://github.com/ansible/ansible-navigator/runs/5065504862?check_suite_focus=true)

```console
FAILED tests/integration/actions/config/test_stdout_tmux.py::Test::test[config specified configuration file without ee  clear && ansible-navigator config dump -c /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/config/ansible.cfg --ee False --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
in #858 

(py3.10 https://github.com/ansible/ansible-navigator/runs/5067847419?check_suite_focus=true)

```console 
FAILED tests/integration/actions/config/test_stdout_tmux.py::Test::test[config specified configuration file without ee  clear && ansible-navigator config dump -c /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/config/ansible.cfg --ee False --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
in #895 

(py 3.7,3.8,3.9 https://github.com/ansible/ansible-navigator/runs/5103067237?check_suite_focus=true)

```
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
in #891 

(py 3.9 3.10 https://github.com/ansible/ansible-navigator/runs/5103072704?check_suite_focus=true)

```console
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
in #874 

(py3.10 https://github.com/ansible/ansible-navigator/runs/5107714816?check_suite_focus=true)

```console
FAILED tests/integration/actions/config/test_stdout_tmux.py::Test::test[config specified configuration file without ee  clear && ansible-navigator config dump -c /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/config/ansible.cfg --ee False --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
in #901 

(py 3.10 https://github.com/ansible/ansible-navigator/runs/5108416365?check_suite_focus=true)

```console
FAILED tests/integration/actions/config/test_stdout_tmux.py::Test::test[config specified configuration file without ee  clear && ansible-navigator config dump -c /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/config/ansible.cfg --ee False --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
in #914 

(py39 https://github.com/ansible/ansible-navigator/runs/5130203887?check_suite_focus=true)
```console
FAILED tests/integration/actions/config/test_stdout_tmux.py::Test::test[config specified configuration file without ee  clear && ansible-navigator config dump -c /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/integration/actions/config/ansible.cfg --ee False --ll debug --mode stdout]
```

**Comment by cidrblock** [comment]
in #935 

(pymany https://github.com/ansible/ansible-navigator/runs/5160687128?check_suite_focus=true)

```console
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee in interactive mode  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode interactive]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee in interactive mode  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode interactive]
```

**Comment by cidrblock** [comment]
in #936 

(py 3.6/3.8 https://github.com/ansible/ansible-navigator/runs/5161634686?check_suite_focus=true)
```console
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee in interactive mode  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode interactive]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee in interactive mode  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode interactive]
```

**Comment by cidrblock** [comment]
in #937 

(py 3.8, 3.9, 3.10 https://github.com/ansible/ansible-navigator/runs/5161824612?check_suite_focus=true)
```console
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode stdout]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment without ee in interactive mode  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee False --ll debug --mode interactive]
FAILED tests/integration/actions/builder/test_stdout_tmux.py::Test::test[build execution-environment with ee in interactive mode  clear && ansible-navigator builder build --tag test_ee --container-runtime docker -v 3 --workdir /home/runner/work/ansible-navigator/ansible-navigator/tests/fixtures/common/builder/test_ee --ee True --ll debug --mode interactive]
=========== 4 failed, 1662 passed, 366 warnings in 625.72s (0:10:25) ===========
ERROR: InvocationError for command /bin/bash -c 'py.test -vv -n `python -c "import multiprocessing; import os; print(int(os.environ.get(\"XDIST_CPU\") or multiprocessing.cpu_count()))"` --dist=loadfile --maxfail=15 --durations=100 --cov --cov-report=xml:/home/runner/work/ansible-navigator/ansible-navigator/.tox/coverage-py38.xml --showlocals ' (exited with code 1)```

**Comment by cidrblock** [comment]
Tests have been stable, can open a new issue when they become unstable
